chirp/drivers/radtel_rt493.py

```diff
@@ -0,0 +1,1152 @@
+# Copyright 2025 Paul Oberosler <paul@paulober.dev>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import time
+import struct
+import logging
+from chirp import bitwise, chirp_common, directory, errors, memmap, util
+from chirp.settings import (
+    RadioSetting,
+    RadioSettings,
+    RadioSettingGroup,
+    RadioSettingValueBoolean,
+    RadioSettingValueList,
+    RadioSettingValueInteger,
+    RadioSettingValueString
+)
+
+
+LOG = logging.getLogger(__name__)
+
+COMMAND_ACCEPT = b"\x06"
+CHARSET_HEX = "0123456789ABCDEFabcdef"
+
+
+def _clean_buffer(radio):
+    radio.pipe.reset_input_buffer()
+    radio.pipe.reset_output_buffer()
+    time.sleep(0.002)
+
+    radio.pipe.timeout = 0.005
+    junk = radio.pipe.read(256)
+    radio.pipe.timeout = 1  # 1000ms
+
+    if junk:
+        LOG.debug("Got %i bytes of junk before starting" % len(junk))
+
+
+def _rawrecv(radio, amount):
+    """Raw read from the radio device"""
+    data = bytearray()
+    try:
+        while len(data) < amount:
+            chunk = radio.pipe.read(amount - len(data))
+            if not chunk:
+                LOG.debug("No response from radio")
+                raise errors.RadioNoContactLikelyK1()
+            data.extend(chunk)
+            time.sleep(0.002)  # 2ms delay
+    except Exception:
```

Comment:
You're raising `RadioNoContactLikelyK1` and then swallowing it here to not show it to the user. If this radio has a (potentially problematic) K1 connector then that's good to have, but you should allow it to actually make to the UI, otherwise it's pointless. You should also `LOG.error` or `LOG.exception` the actual failure in order to be useful for debugging.